src/scripts/utils.js

```diff
@@ -7,3 +7,13 @@
 export const getHexString = input => {
   return input.toString('hex');
 };
+
+/**
+ * Convert BTC to Satoshi.
+ *
+ * @param btc btc
+ * @returns satoshi
+ */
+export const toSatoshi = btc => {
+  return btc / 0.00000001;
```

Comment:
What about:
```suggestion
  return btc * 100000000;
```

It is a little more elegant and I have had a rounding error when testing this PR.